### ISSUE TITLE
'HTMLVideoElement.webkitExitFullscreen()' is deprecated

### ISSUE THREAD

**Issue by A-Yamout** [FinishedManx843/Membean_Bot#2]
'HTMLVideoElement.webkitExitFullscreen()' is deprecated. You might want to use 'Document.exitFullscreen()'

**Comment by A-Yamout** [comment]
@FinishedManx843

**Comment by Sluke950** [comment]
I don't remember using any exit full screen. I will look into it but its been a while since ive done any JavaScript. thanks for the heads up!

**Comment by A-Yamout** [comment]
NP

**Comment by EZawesome** [comment]
@FinishedManx843 yeah I think I have the same issue if you could/had time to fix it, it would be very helpful. Thank you.
